dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/src/org/apache/cxf/common/logging/CXFLogger.java

```diff
@@ -0,0 +1,180 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ * 
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package org.apache.cxf.common.logging;
```

Comment:
Since this not a CXF class, we should move this package to `com.ibm.ws.cxf.logging` to match the name of our internal APIs in CXF bundles. That way it easily distinguishable from overridden CXF code.

Reply:
actually, all new packages should be `io.openliberty` rather than `com.ibm.ws`, but I agree, non overlay code should be moved to `io.openliberty.cxf.logging`